tyk-pro/templates/deployment-gw-repset.yaml

```diff
@@ -132,6 +132,63 @@ spec:
         {{- end }}
           - name: TYK_GW_HTTPSERVEROPTIONS_USESSL
             value: "{{ .Values.gateway.tls }}"
+          # Default Settings
+          - name: TYK_GW_TEMPLATEPATH
+            value: "/opt/tyk-gateway/templates"
+          - name: TYK_GW_TYKJSPATH
+            value: "/opt/tyk-gateway/js/tyk.js"
+          - name: TYK_GW_MIDDLEWAREPATH
+            value: "/mnt/tyk-gateway/middleware"
+          - name: TYK_GW_USEDBAPPCONFIGS
+            value: "true"
+          - name: TYK_GW_APPPATH
+            value: "/mnt/tyk-gateway/apps"
+          - name: TYK_GW_ENABLEANALYTICS
+            value: "true"
+          - name: TYK_GW_ANALYTICSCONFIG_TYPE
+            value: "mongo"
```

Comment:
I'm not sure, but I want to ask something. According to docs, `analytics_config.type` can be set to empty string or `rpc`. Is there any reason why we are setting it to `mongo`? I know that it is defined as `mongo` in the previous mgmt.conf file.

Reply:
Nice catch! Gateway doesn't even talk to mongo. I will set it to empty.